tests/aws/test_client.py

```
@@ -238,7 +247,7 @@ def test_invite_organization_to_transfer_billing(
     )
 
 
-def test_get_responsibility_transfer_details(config, aws_client_factory, mock_get_paged_response):
+def test_get_transfer_details(config, aws_client_factory, mock_get_paged_response):
```

Ruff (0.14.13)

Unused function argument: `mock_get_paged_response`

(ARG001)